README.md

```diff
@@ -1 +1,238 @@
 # Sync and Async
+
+## Overview
```

Comment:
This overview is pretty confusing, and I'm not sure what problem it's presenting. At the risk of echoing @vicfriedman too much, we need to tell students about a problem ("sometimes we need to perform expensive operations that block") and how JavaScript's (specifically, V8's) event loop can help solve it ("we put everything on a single thread but execute in a non-blocking way").

Obviously the above is a bit too technical, but it helps focus the issue.